hazelcast/src/hazelcast/client/network.cpp

```diff
@@ -64,1271 +64,1639 @@
 #include "hazelcast/client/connection/wait_strategy.h"
 
 namespace hazelcast {
-    namespace client {
-        namespace connection {
-            constexpr size_t ClientConnectionManagerImpl::EXECUTOR_CORE_POOL_SIZE;
-            const endpoint_qualifier ClientConnectionManagerImpl::PUBLIC_ENDPOINT_QUALIFIER{CLIENT, "public"};
-
-            ClientConnectionManagerImpl::ClientConnectionManagerImpl(spi::ClientContext &client,
-                                                                     std::unique_ptr<AddressProvider> address_provider)
-                    : alive_(false), logger_(client.get_logger()),
-                      connection_timeout_millis_((std::chrono::milliseconds::max) ()),
-                      client_(client), socket_interceptor_(client.get_client_config().get_socket_interceptor()),
-                      address_provider_(std::move(address_provider)), connection_id_gen_(0),
-                      heartbeat_(client, *this),
-                      async_start_(client.get_client_config().get_connection_strategy_config().is_async_start()),
-                      reconnect_mode_(client.get_client_config().get_connection_strategy_config().get_reconnect_mode()),
-                      smart_routing_enabled_(client.get_client_config().get_network_config().is_smart_routing()),
-                      client_uuid_(client.random_uuid()),
-                      authentication_timeout_(boost::chrono::milliseconds(heartbeat_.get_heartbeat_timeout().count())),
-                      load_balancer_(client.get_client_config().get_load_balancer()),
-                      wait_strategy_(client.get_client_config().get_connection_strategy_config().get_retry_config(),
-                                     logger_), cluster_id_(boost::uuids::nil_uuid()),
-                      connect_to_cluster_task_submitted_(false),
-                      use_public_address_(address_provider_->is_default_provider() &&
-                                          client.get_client_config().get_network_config().use_public_address()) {
-                config::client_network_config &networkConfig = client.get_client_config().get_network_config();
-                auto connTimeout = networkConfig.get_connection_timeout();
-                if (connTimeout.count() > 0) {
-                    connection_timeout_millis_ = std::chrono::milliseconds(connTimeout);
-                }
+namespace client {
+namespace connection {
+constexpr size_t ClientConnectionManagerImpl::EXECUTOR_CORE_POOL_SIZE;
+const endpoint_qualifier ClientConnectionManagerImpl::PUBLIC_ENDPOINT_QUALIFIER{
+    CLIENT,
+    "public"
+};
+
+ClientConnectionManagerImpl::ClientConnectionManagerImpl(
+  spi::ClientContext& client,
+  std::unique_ptr<AddressProvider> address_provider)
+  : alive_(false)
+  , logger_(client.get_logger())
+  , connection_timeout_millis_((std::chrono::milliseconds::max)())
```

Comment:
do we really want these `,` characters at the beginning?

Reply:
What do you suggest?

Reply:
This the Mozilla's default and personally, I like it. Every member is on a different line, so it is easier the see what are the members and how are they being initialized. The commas at the beginning makes everything nicely aligned with the initial line starting with a colon. I agree that it might look a bit weird at first but once you've seen a few, it's clear.

Please note that you don't have to write your code like this. You just write however you like and press some shortcut on your IDE and it will make it this way.